syntax = "proto3";

package protocol_msg;

message LoginMsg {
	int32 msg_type = 1;
	int32 timestamp = 2;
	string user_id = 3;
	string password = 4;
	int32 authority = 5;
}